authors/apps/articles/serializers.py

```diff
@@ -105,4 +105,36 @@ def validate(self, data):
         data["article"] = self.article
         data["rated_by"] = user
 
-        return data
\ No newline at end of file
+        return data
+
+
+class LikeSerializer(serializers.ModelSerializer):
+    action_performed = "created"
+
+    class Meta:
+        model = Like
+        fields = ['user', 'article', 'like']
+
+    def create(self, validated_data):
+        found = False
+        try:
+            self.instance = Like.objects.filter(article=validated_data["article"].id, user=validated_data["user"].id)[0:1].get()
+            found = True
+        except Like.DoesNotExist:
```

Comment:
@byarustev , I think you should just return the like object here, instead of putting `pass`. The `if` statement below is unneccessary.